HW3/03_2_ Practical_tasks_GitHub.py

```diff
@@ -0,0 +1,35 @@
+def analyze_number(number):
+    """Аналізує чотиризначне число та виводить результати."""
+
+    # Перевірка, чи число чотиризначне
+    if not (1000 <= number <= 9999):
+        print("Будь ласка, введіть чотиризначне число.")
+        return
+
+    # Перетворення числа на рядок для зручності роботи з цифрами
+    number_str = str(number)
+
+    # Обчислення добутку цифр
+    product = 1
+    for digit in number_str:
```

Comment:
look at this version of code without loop
number = input("Enter a four digit number:")
print(f'The product of numbers is: {int(number[0]) * int(number[1]) * int(number[2]) * int(number[3])}')

or this version

number = int(input("Enter four digit number: "))

num_1 = number // 1000
num_2 = (number // 100) % 10
num_3 = (number // 10) % 10
num_4 = number % 10

product = num_1 * num_2 * num_3 * num_4
print(product)